### ISSUE TITLE
Nested attribute translation in a translation not working. 

### ISSUE THREAD

**Issue by phoenixking25** [kubernetes/dashboard#3650]
##### Environment

```
Installation method:
Kubernetes version:
Dashboard version:v1.10.0
Operating system:  Ubuntu 18.04
Node.js version ('node --version' output): v11.10.0
Go version ('go version' output): go1.11.4
```

##### Steps to reproduce

`ng xi18n --out-file source.xlf`
##### Observed result

Similar issue as observed. https://github.com/angular/angular/issues/13796
`Could not mark an element as translatable inside a translatable section ("mage's default entrypoint command. You can use the command options to override the default.
       [ERROR ->]<a href="http://kubernetes.io/docs/user-guide/containers/"
          target="_blank"
          tabi"): /home/phoenix/go/src/github.com/kubernetes/dashboard/src/app/frontend/create/from/form/template.html@295:8`
##### Expected result

Successfully generated translation file.
##### Proposed Solution

We can make a different component for the `Learn more Hyperlink Button` for the docs reference so we can escape the nested attribute translation. It is yet to be experimented :P that it works in that case or not, but it should work perfectly from my instinct.
We have this type of errors only in Learn more hyperlinks so we need only one component and we can reduce length of code that is repeating in `template.html`  

Related to #3574

**Comment by phoenixking25** [comment]
@maciaszczykm

**Comment by feloy** [comment]
I think this issue were fixed by #3770
